### DISCUSSION TITLE
Looking to hire someone to help me create .ASS template files

### DISCUSSION THREAD

**Discussion by lightninglu10** [libass/libass#720] [General]
Hey all, we're looking for someone to help us create 10-15 template files with motion text in .ASS format.

Does anyone here do that kind of work, or does anyone know someone who does this work? 

We're looking for people experienced in this format.

**Comment by witchymary** [comment]
By template, do you mean kara templates? Or just overall typesetting?

**Reply by lightninglu10** [reply]
Sorry, I'm a bit new to this file type, what are Kara templates?

But overall type setting with motion based text. If you send me an email that's listed on my Github we can talk more

**Comment by TheOneric** [comment]
@lightninglu10 if you found someone, it’d be good to close this discussion. If you haven’t yet, it would probably be better to publicly clarify here what exactly you are looking for. *(Though this is not a recruitment platform.)*

**Comment by lightninglu10** [comment]
Hey @TheOneric we haven't found anyone yet. 

Basically we're looking for someone to help us build some animated .ASS files, something like this: https://youtube.com/shorts/Xd9IeCODKG4?si=_r0sqYazaH76tdxN

or this: https://youtu.be/9ngNySFSWWU?si=yda4l8qdmCF8ojBt

The YouTube examples above are a bit complex, so it doesn't need to be that complex, but something animated. We have an app https://www.autocpt.com/ that utilizes .ASS templates to create subtitles and captions on videos and would like to work with someone to help us build out animated templates that our users can apply to their video captions.
